controllers/profiles/common/properties/managed_test.go

```diff
@@ -121,11 +121,11 @@ func Test_appPropertyHandler_WithUserPropertiesWithNoUserOverrides(t *testing.T)
 	//just add some user provided properties, no overrides.
 	userProperties := "property1=value1\nproperty2=value2"
 	workflow := test.GetBaseSonataFlow("default")
-	props, err := NewAppPropertyHandler(workflow, nil)
+	props, err := NewManagedPropertyHandler(workflow, nil)
 	assert.NoError(t, err)
 	generatedProps, propsErr := properties.LoadString(props.WithUserProperties(userProperties).Build())
 	assert.NoError(t, propsErr)
-	assert.Equal(t, 6, len(generatedProps.Keys()))
+	assert.Equal(t, 7, len(generatedProps.Keys()))
```

Comment:
sorry for the silly question, but why  are we now expecting 7 properties instead of 6?
a new by default property was introduced?

Reply:
Now we are adding the knative health check prop by default. Before, Knative properties were only set if the platform was passed to the creator, which doesn't fit the use case.